app/(main)/_components/_prizes/prizes-section.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import Image from "next/image";
+
+import Prize from "@/app/(main)/_components/_prizes/prize";
+
+const PrizesSection = () => {
+  const [pantherHovered, setPantherHovered] = useState(false);
+  return (
+    <div
+      id="prizes"
+      className="relative flex flex-col w-full items-center justify-end gap-y-8 sm:gap-y-6 md:gap-y-4 pt-40 px-8 md:px-20 lg:px-32 overflow-x-clip"
+    >
+      <div className="flex flex-col items-center justify-center text-center gap-y-4">
+        <h2 className="font-TangoSansBold text-6xl  text-primary pb-10">Prizes</h2>
+        <p>One per winning team member!</p>
```

Comment:
I think this text relating to this section should be something more helpful for the section as a whole. An example could be the following:
```suggestion
        <p>Learn more about the prizes we offer.</p>
```

Reply:
I agree! Committing this as well.